chart/jenkins-operator/templates/cert-manager.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: webhook-certificate
+  namespace: {{ .Values.jenkins.namespace }}
```

```suggestion
  namespace: {{ .Release.Namespace }}
```
Webhooks should be used per Operator, not per Jenkins and we recommend deploying Operator in separate namespace from Jenkins for better security :)